plugins/board-resources/src/components/popups/CopyCard.svelte

```diff
@@ -65,6 +75,83 @@
       value,
       newCardId
     )
+
+    if (keepAttachments) {
+      const sourceAttachments = await client.findAll(attachment.class.Attachment, {
+        space: object.space,
+        attachedTo: object._id
+      })
+
+      for (let i = 0; i < sourceAttachments.total; i++) {
+        const a = sourceAttachments[i]
+
+        await client.addCollection(
```

Comment:
We have big trouble here, we need to have API to copy or increment reference on S3/minio for particular file, since it could be deleted if original card will be deleted, since Attachment object is just a descriptor for a real document inside file storage. Let's discuss on our general meeting.